scripts/core/editor3/directive.tsx

```diff
@@ -245,6 +246,7 @@ class Editor3Directive {
                                     cleanPastedHtml={this.cleanPastedHtml}
                                     autocompleteSuggestions={autocompleteSuggestions}
                                     plainText={this.plainText}
+                                    canAddArticleEmbed={(srcId: string) => canAddArticleEmbed(srcId, this.item._id)}
```

Comment:
Doesn't type inference work here?

Reply:
It doesn't. Because of multiple levels of wrapping.